test.js

```diff
@@ -1,23 +1,23 @@
 import test from 'ava'
 
-import identifierfy from './'
+import identifierfy from '.'
```

Comment:
small q: would you prefer to use CommonJS here too so it's consistent with index.js?